Nio/Conversations/Event Views/MessageView/BorderlessMessageView.swift

```diff
@@ -25,10 +34,24 @@ struct BorderlessMessageView<Model>: View where Model: MessageViewModelProtocol
         .font(.system(size: 60 * sizeCategory.scalingFactor))
     }
 
+    var editedView: some View {
+        Text("(" + L10n.Event.edit + ")")
+            .font(.caption)
+            .foregroundColor(textColor)
+            .opacity(colorSchemeContrast == .standard ? 0.5 : 1.0)
+    }
+
     var contentView: some View {
         emojiView
     }
 
+    var contentEditedView: some View {
+        VStack {
+            emojiView
+            editedView
+        }
```

Comment:
What do you think about making this an `HStack` to mirror Riot's UI? It does however suck to have to change the order based on `isMe` 🤦‍♀️.

```suggestion
        HStack(alignment: .bottom) {
            if isMe {
                editedView
                emojiView
            } else {
                emojiView
                editedView
            }
        }
```

Reply:
Sure let's do that